common/src/configuration/network.rs

```diff
@@ -86,6 +95,66 @@ impl Network {
             LocalNet => "localnet",
         }
     }
+
+    /// This function returns the network wire byte for any chosen network. Increase these numbers for any given network
+    /// when network traffic separation is required.
+    /// Note: Do not re-use previous values.
+    pub fn as_wire_byte(self) -> u8 {
+        let wire_byte = match self {
+            // Choose a value in 'MAIN_NET_RANGE' or assign 'self.as_byte()'
+            Network::MainNet => self.as_byte(),
+            // Choose a value in 'STAGE_NET_RANGE' or assign 'self.as_byte()'
+            Network::StageNet => self.as_byte(),
+            // Choose a value in 'NEXT_NET_RANGE' or assign 'self.as_byte()'
+            Network::NextNet => self.as_byte(),
+            // Choose a value in 'LOCAL_NET_RANGE' or assign 'self.as_byte()'
+            Network::LocalNet => self.as_byte(),
+            // Choose a value in 'IGOR_RANGE' or assign 'self.as_byte()'
+            Network::Igor => self.as_byte(),
+            // Choose a value in 'ESMERALDA_RANGE' or assign 'self.as_byte()'
+            Network::Esmeralda => self.as_byte(),
+        };
+        // The error is squashed to make this method infallible, and the method is called here to ensure that the
+        // range constants are used.
+        let _unused = self.verify_network_wire_byte_range(wire_byte);
```

Comment:
This does not need to happen at all - the `verify_network_wire_byte_range`  function can become a test that runs.

Reply:
```rust
#[test]
fn check_constant_wire_bytes_are_in_correct_range() {
verify_network_wire_byte_range(Network::Esmeralda);
// etc..
}

 fn verify_network_wire_byte_range(self, network: Network) -> Result<(), String> {
        
// 'LIVENESS_WIRE_MODE' is reserved for '0xa7'
        if network_wire_byte == LIVENESS_WIRE_MODE {
            panic!("Invalid network wire byte, cannot be '0x46', reserved for 'LIVENESS_WIRE_MODE'");
        }

        // Verify binned values
        let valid = match self {
            Network::MainNet => MAIN_NET_RANGE.contains(&network_wire_byte),
            Network::StageNet => STAGE_NET_RANGE.contains(&network_wire_byte),
            Network::NextNet => NEXT_NET_RANGE.contains(&network_wire_byte),
            Network::LocalNet => LOCAL_NET_RANGE.contains(&network_wire_byte),
            Network::Igor => IGOR_RANGE.contains(&network_wire_byte),
            Network::Esmeralda => ESMERALDA_RANGE.contains(&network_wire_byte),
        };
        if !valid {
          panic!(
                "Invalid network wire byte `{}` for network `{}`",
                network_wire_byte, network
            );
        }
    }